### PR TITLE
Improves first-time experience by switching default to MySQL

### LINKED ISSUE #61: Switch default storage to MySQL
I think it is time we switch the default docker-compose file to mysql. The goal of docker-compose was our 5 minute story. MySQL is the fastest way to tell the story of zipkin, particularly as it has no setup required to show the Dependencies view.

I'm glad we have cassandra, as people may quickly outgrow mysql, but I feel we should put easiest as the first option forward.

### PR DESCRIPTION
**Pull request by adriancole**
MySQL is the best environment for people to learn zipkin. It is widely
understood, can be easily explored using SQL, supports on-demand
dependency aggregation, returns precise results for queries, and has
had very few support issues.

While we want to show people how to work with Cassandra at some point,
it has proven a high barrier entry for the opposite of above. People
rarely know cassandra intimately, it has imprecise results (usually
users have to over-fetch), the duration query doesn't work, the
dependency graph requires a cron, and it has had numerous support
issues.

The point of our docker-compose file is to ease the 5 minute, getting
started story of Zipkin. At the moment, MySQL is by far the best choice
for this and is now the default.

Fixes #61

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
:shipit: - makes perfect sense

**Comment by anuraaga** [comment]
Anything is simpler than Cassandra so seems reasonable :) 

I'd make more explicit in zipkin-docker that the compose is primarily a playground, I didn't quite get that feeling when reading. I guess the idea is the default is to get a quick and dirty collector and Zipkin UI up quickly to feel it out but is not recommended for production.

**Comment by adriancole** [comment]
good point about being more explicit wrt docker-compose being a toy. We
should somehow say docker images for zipkin are intended for prod, but the
others are toys and docker-compose is also a toy.

**Comment by mansu** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -10,11 +10,31 @@
 
 This repository contains the Docker build definition and release process for
 [openzipkin/zipkin](https://github.com/openzipkin/zipkin). It also contains
-test images for transport and storage backends such as Cassandra.
+test images for transport and storage backends such as Kafka or Cassandra.
 
 Automatically built images are available on Quay.io under the [OpenZipkin](https://quay.io/organization/openzipkin) organization,
 and are mirrored to [Docker Hub](https://hub.docker.com/u/openzipkin/).
 
+## Regarding production usage
```
Comment (adriancole) [inline README.md]: @anuraaga heavy disclaimers here :) cc @abesto

File: docker-compose.yml [hunk 2]
```diff
@@ -1,31 +1,36 @@
 # This file uses the version 2 docker-compose file format, described here:
 # https://docs.docker.com/compose/compose-file/#version-2
 #
-# It runs the zipkin and zipkin-cassandra containers, using docker-compose's
+# This runs the zipkin and zipkin-mysql containers, using docker-compose's
 # default networking to wire the containers together.
+#
+# Note that this file is meant for learning Zipkin, not production deployments.
```
Comment (adriancole) [inline docker-compose.yml]: just in case.. added another note here :)

**Comment by anuraaga** [comment]
LGTM

**Comment by adriancole** [comment]
verified in mysql the spark job eventually runs, too. (mysql computes on demand until the spark job is run)

```
dependencies    | Running Dependencies job for 2016-09-09: start_ts between 1473379200000000 and 1473465599999999
dependencies    | Running Dependencies job for 2016-09-08: start_ts between 1473292800000000 and 1473379199999999
dependencies    | 02:00:02.056 [main] WARN  o.a.hadoop.util.NativeCodeLoader - Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
dependencies    | 02:00:02.111 [main] WARN  o.a.hadoop.util.NativeCodeLoader - Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
dependencies    | Dependencies: []
dependencies    | Dependencies: [{"parent":"fakewebapi","child":"fakewebapi","callCount":2}]
```

**Comment by jcarres-mdsol** [comment]
I'm late but great idea, +1

**Pull request merged**